perl/README.md

```diff
@@ -0,0 +1,1028 @@
+| chr20 | 1110696 | MULTIALLELIC | A        | G   | 0    | NA00001;NA00002 | 0.667          | NA          | 0            | NA           | 0           | 2   | 6   | 0.333     | 1110696 | MULTIALLELIC_SNP            |
+| chr20 | 1110696 | MULTIALLELIC | A        | T   | 0    | NA00001;NA00002 | 0.667          | NA00003     | 0.333        | NA           | 0           | 4   | 6   | 0.667     | 1110696 | MULTIALLELIC_SNP            |
+| chr20 | 1       | INS          | A        | +C  | 0    | NA00003         | 0.333          | NA          | 0            | NA           | 0           | 1   | 6   | 0.167     | 1       | SIMPLE_INSERTION            |
+| chr20 | 1       | INS          | A        | +CC | 0    | NA00001;NA00002 | 0.667          | NA00003     | 0.333        | NA           | 0           | 4   | 6   | 0.667     | 1       | INSERTION_BETWEEN_TWO_BASES |
+| chr20 | 1234568 | MULTIALLELIC | T        | -3  | 0    | NA00001         | 0.333          | NA00003     | 0.333        | NA           | 0           | 3   | 6   | 0.5       | 1234567 | microsat1                   |
+| chr20 | 1234568 | MULTIALLELIC | T        | +A  | 0    | NA00002         | 0.333          | NA          | 0            | NA           | 0           | 1   | 6   | 0.167     | 1234567 | microsat1                   |
+| chr20 | 3       | MNP          | C        | A   | 2    | NA              | 0              | NA00003     | 0.5          | NA00001      | 0.333       | 2   | 4   | 0.5       | 3       | EXAMPLE_MISSING_MNP         |
+| chr20 | 4       | MNP          | C        | A   | 2    | NA              | 0              | NA00003     | 0.5          | NA00001      | 0.333       | 2   | 4   | 0.5       | 3       | EXAMPLE_MISSING_MNP         |
+| chr20 | 5       | MNP          | C        | A   | 2    | NA              | 0              | NA00003     | 0.5          | NA00001      | 0.333       | 2   | 4   | 0.5       | 3       | EXAMPLE_MISSING_MNP         |
+
+Expected Bystro Genotype Dosage Matrix Output:
+
+```python
+import pandas as pd
+df = pd.read_feather('example_vcf_dosage_matrix.feather')
+print(df)
+```
+
+| locus              | NA0001 | NA0002 | NA0003 |
+| ------------------ | ------ | ------ | ------ |
+| chr20:1:A:T        | 1      | 0      | 0      |
+| chr20:1110696:A:G  | 1      | 1      | 0      |
+| chr20:1110696:A:T  | 1      | 1      | 2      |
+| chr20:1:A:+C       | 0      | 0      | 1      |
+| chr20:1:A:+CC      | 1      | 1      | 2      |
+| chr20:1234568:T:-3 | 1      | 0      | 2      |
+| chr20:1234568:T:+A | 0      | 1      | 0      |
+| chr20:3:C:A        | -1     | 0      | 2      |
+| chr20:4:C:A        | -1     | 0      | 2      |
+| chr20:5:C:A        | -1     | 0      | 2      |
+
+- Note that missing genotypes are reprsented as -1 in the genotype dosage matrix output
+- If a sample's genotype contains any missing genotypes, the sample is considered missing for the site
```

Comment:
I feel like this sentence is a little confusing. It feels like it's saying if a sample is missing at a site, the sample will be missing at a site. Let me know if it's supposed to mean something else.

Reply:
Should be clearer now?